extras/buildah/build.py

```diff
@@ -0,0 +1,113 @@
+#!/usr/bin/env python3
+#
+# Build heketi containers using the 'buildah' tool either from source
+# or from existing compiled binaries.
+#
+
+import argparse
+import os
+import shutil
+import subprocess
+import tempfile
+
+USE_SUDO = True
+
+pjoin = os.path.join
+
+
+def buildah(*args, redirect=False):
+    """A lightweight wrapper around buildah command."""
+    cmd = []
+    if USE_SUDO and os.getuid() != 0:
+        cmd.append("sudo")
+    cmd.append("buildah")
+    cmd.extend(args)
+    if redirect:
+        return subprocess.check_output(cmd).strip().decode('utf8')
+    else:
+        subprocess.check_call(cmd)
+
+
+def buildah_from(src):
+    return buildah('from', src, redirect=True)
+
+
+def build_binaries(outdir):
+    pkgs = 'glide golang git make mercurial'
+    hdir = '/build/src/github.com/heketi/heketi'
+    heketi_branch = 'master'
+    heketi_url = 'https://github.com/heketi/heketi.git'
+
+    bc = buildah_from('fedora:27')
```

Comment:
I assume `fedora:27` to get a functional version of glide? A comment and maybe reference to the BZ would be  good.

Reply:
Yes, good idea.